<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ESM</class>
 <widget class="QMainWindow" name="ESM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ESM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>171</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="video1">
       <property name="text">
        <string>Video 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Q1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video2">
       <property name="text">
        <string>Video 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scan">
       <property name="text">
        <string>Scan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video3">
       <property name="text">
        <string>Video 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exploration">
       <property name="text">
        <string>Exploration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video4">
       <property name="text">
        <string>Video 4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Q2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video5">
       <property name="text">
        <string>Video 5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tests">
       <property name="text">
        <string>Tests</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="video6">
       <property name="text">
        <string>Video 6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>QF</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="end">
       <property name="text">
        <string>FIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
